test/dlc_tests/benchmark/ec2/tensorflow/inference/test_performance_tensorflow_inference.py

```diff
@@ -72,7 +72,11 @@ def ec2_performance_tensorflow_inference(image_uri, processor, ec2_connection, e
     tf_version = "1" if is_tf_version("1", image_uri) else "2"
     _, tf_api_version = get_framework_and_version_from_tag(image_uri)
 
-    num_iterations = 500 if is_pr_context() else 1000
+    # setting 1000 iterations on graviton causes the test to timeout thru codebuild, this does not
+    # happen when the test is ran manually. Setting graviton to same iteration as PR.
+    # TODO: revamp test supporting same configurations on x86_64 and graviton
+    num_iterations = 500 if is_pr_context() or "graviton" in image_uri else 1000
```

Comment:
Can we add a function (is_gravition_image()) that returns true if image_uri is a graviton image and false when it is not? That's preferable to the several inline checks with the string "graviton" in this function; and maybe elsewhere. This will limit future code changes to just the function, and especially if the criteria for validating a graviton image changes.

Reply:
created bool variable. same setup as [here](https://github.com/aws/deep-learning-containers/blob/master/test/dlc_tests/ec2/tensorflow/inference/test_tensorflow_inference.py#L118). As I'm not expecting this to be here long. This setup should be viable for this need.